crates/types/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "rjacraft_types"
+description = "Types for rjacraft"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+keywords.workspace = true
+categories.workspace = true
+homepage.workspace = true
+repository.workspace = true
```

Comment:
Specify the categories too. Which ones should these be, however?

Reply:
I think `api-bindings` will be the most appropriate. The generator package can take `development-tools::build-utils`.

Reply:
As for the tags, I think that `minecraft` will suffice.